setup.py

```diff
@@ -50,8 +50,10 @@ def _get_package_version():
     install_requires=[
         # In order to control dbt-core version and package version
         "boto3~=1.26",
+        "boto3-stubs[athena,glue,lakeformation,sts]~=1.26",
         "dbt-core~=1.4.5",
         "pyathena~=2.24",
+        "pydantic~=1.10",
```

Comment:
About pydantic, I am not against having it as a dependency but for our use case, the plain and simple dataclasses are not sufficient ?

Reply:
Unfortunately not. Dataclasses does not support nested structures by default, that's why we need pydantic

Reply:
By nested structures you mean for this:
```python
filters: Dict[str, FilterConfig]
``` 
?

Reply:
Indeed I tested and I thought that dataclasses were working a little bit better than that 😅 .
It seems ok for me to use Pydantic for this case 👍🏻